### ISSUE TITLE
Proposition d'alarmes intelligentes

### ISSUE THREAD

**Issue by notaustil** [natinusala/openedt#2]
En tant qu'étudiants nos journées sont rythmées par notre emploi du temps. Celui ci varie et l'heure à laquelle on se lève le matin en dépend (très probablement :smiley:).

L'idée serai que l'application proposerais le soir, via une notification par exemple, d'activer l'alarme la plus appropriée en fonction de l'heure du premier cours du lendemain.

Qu'en pensez vous ? Killer feature ou pas ?

**Comment by maveist** [comment]
Oui, mais cependant pour la notification propose une heure qui coïncide avec l'heure de réveil de l'utilisateur, il faudrait que l'utilisateur paramètre tout ça justement.
Par exemple, l'utilisateur  paramètre l'application pour qu'il propose le réveil à 7h lorsque ses cours commence à 8h.

Sinon j'aime bien l'idée! :D

**Comment by natinusala** [comment]
L'idée est très intéressante :) Il faudrait pouvoir le personnaliser avec des options comme "combien de temps mettez-vous à vous préparer et vous rendre à l'université" ou "de combien de temps voudriez-vous arriver en avance", pour le rendre le plus souple possible.

**Comment by maveist** [comment]
Cela rejoins, l'idée de paramètrage. :)
